tools/gromacs/macros.xml

```diff
@@ -24,6 +24,15 @@
                 <param argument="mdp_input" type="data" format='mdp' label="MD parameters (MDP) file (optional; default settings if not set)"/>
             </when>
             <when value="default">
+                <conditional name="startingvelocities">
+                   <param name="startvel" type="select" label="Starting Velocities">
+                       <option value="true">Assign starting velocities (recommended for NVT)</option>
+                       <option value="false" selected="true">Do not assign starting velocites (recommended for NPT)</option>
+                   </param>
+                     <when value="true"/>
+                     <when value="false"/>
+                </conditional>
```

Comment:
```suggestion
                <param name="startvel" type="select" label="Starting Velocities">
                    <option value="true">Assign starting velocities (recommended for NVT)</option>
                    <option value="false" selected="true">Do not assign starting velocites (recommended for NPT)</option>
                </param>
```
As far as I can see you do not need a conditional here, you only need this if you want to show different params based on the conditional, which can be specified under `<when/>`.

Reply:
I am removing this from macros.xml, and instead adding this option in a new conditional in sim.xml (ensembleselect), when selecting the ensemble. This will make sure the option to assign velocities only comes up when a user selects NVT as the ensemble, so its less confusing.